index.html

```diff
@@ -135,21 +150,26 @@ <h6 class="client">Canopy</h6>
             </div>
         </div>
         <div class="about-myself">
-            <h2 class= "title-about-myself">About myself</h2>
-            <p class="intro">I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
-            <h6 class="connect">let's connect</h6>
-            <ul class="networks-about">
-                <li><a href=""><img src="img/GitHub.png" alt="20px"></a></li>
-                <li ><a href=""><img src="img/Linkedin icon.png" alt=""></a></li>
-                <li><a href=""><img src="img/Peace.png" alt="20px"></a></li>
-                <li><a href=""><img src="img/Twitter.png" alt="20px"></a></li>
-                <li><a href=""><img src="img/M.png" alt="20px"></a></li>
-            </ul>
-            <div class="about-action">
-                <button type="button" disabled class="btn normal-b">Get my resume</button>
-            </div>
+            <div class="left-block-about">
+                <h2 class= "title-about-myself">About myself</h2>
+                <p class="intro-about intro">I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
+                <h6 class="connect-about connect">let's connect</h6>
+                <ul class="networks-about">
+                    <li><a href=""><img src="img/GitHub.png" alt="20px"></a></li>
+                    <li ><a href=""><img src="img/Linkedin icon.png" alt=""></a></li>
+                    <li><a href=""><img src="img/Peace.png" alt="20px"></a></li>
+                    <li><a href=""><img src="img/Twitter.png" alt="20px"></a></li>
+                    <li><a href=""><img src="img/M.png" alt="20px"></a></li>
+                </ul>
+                <div class="about-action">
+                    <button type="button" disabled class="btn normal-b">Get my resume</button>
+                </div>
```

Comment:
- [ ] Kindly try to match the Figma design for the `Get My Resume` button. :+1: 

Your design             |  Figma Design
:-------------------------:|:-------------------------:
![design_get-my](https://user-images.githubusercontent.com/70483835/221090765-5d929ee5-9922-43ea-b3f7-b3f32d3fb265.png) | ![figma_get_in_touch](https://user-images.githubusercontent.com/70483835/221090829-69e4d41b-b1a9-4bec-adde-5b0476303a1d.png)